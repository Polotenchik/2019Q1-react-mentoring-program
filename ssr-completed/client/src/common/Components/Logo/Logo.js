import React from 'react';
import styled from 'styled-components';
import Theme from '../../Theme/Theme';

const Logo = () => <LogoText>netflixroulette</LogoText>;

export default Logo;

const LogoText = styled.a`
    color: ${Theme.colors.red};
    font-size: 16px;
`;
