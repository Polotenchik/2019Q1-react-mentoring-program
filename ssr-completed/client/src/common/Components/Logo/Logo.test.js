import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo Snapshot', () => {
    it('Logo Component should render correctly with no props', () => {
        const component = shallow(<Logo />);
        expect(component).toMatchSnapshot();
    });
});