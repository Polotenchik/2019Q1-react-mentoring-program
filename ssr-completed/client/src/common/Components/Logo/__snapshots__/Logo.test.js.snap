// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Snapshot Logo Component should render correctly with no props 1`] = `
<styled.a>
  netflixroulette
</styled.a>
`;
